test/e2e/commit_test.go

```diff
@@ -225,7 +225,7 @@ var _ = Describe("Podman commit", func() {
 		Expect(inspect).Should(Exit(0))
 		image := inspect.InspectImageJSON()
 		_, ok := image[0].Config.Volumes["/foo"]
-		Expect(ok).To(BeTrue())
+		Expect(ok).To(BeTrue(), ".Config.Volumes[/foo]")
```

Comment:
FYI  This can be changed to `Expect(image[0].Config.Volumes).To(HaveKey("/foo))`

The same for the other place where you compare a map. There is also a `HaveKeyWithValue()` function.

Reply:
Oh, nice, thank you!